### ISSUE TITLE
Xfce fails and force quits Termux

### ISSUE THREAD

**Issue by ihtarlik** [LinuxDroidMaster/Termux-Desktops#63]
I am running a Samsung SM-S146V (CPU: MT6833V/NZA (8), Memory: 2685MiB / 3608) with Android 14, and the latest builds of Termux and Termux-X11.

I just refreshed my proot-distro copy of Debian and installed Xfce per the instructions in this project. When I run the X11 launch script, it opens Termux-X11, and I briefly get a desktop, before it goes completely black in Termux-X11. When I swipe over to the Termux window, it has entirely exited and reopened Termux.

I have tried other desktops, and they aren't having this problem. I am able to run LXDE with audio and OpenGL rendering.

The only system modifications I make after the base image is to run:

apt install nmap nano unzip wget sudo

And to add a non-root user with sudo permissions. I also can't find any logs for Xfce to be able to troubleshoot the problem.

**Comment by ihtarlik** [comment]
Cinnamon does the same thing. I guess LXDE is the only one that works for me currently.

**Comment by anonimo82** [comment]
Please, can you check `Ubuntu` in `Proot-Distro` and tell me if `Blender` works with hardware acceleration?
Version should be `4.0.1`.
I'm just curious, if it works, I could consider upgrading my device.
And if you succeed, please, post here what you did.

Thanks in advance for either, regards, have a nice day.

**Comment by ihtarlik** [comment]
I attempted to run Ubuntu with Gnome desktop, and it took too long to load on my device.

Also, this comment has nothing to do with the original issue posted about various DEBIAN desktops not working properly.

**Comment by anonimo82** [comment]
Yes, I know. 
It would be a personal favor to me, an unknown friend on the web.
If you have time and will, please, try with `apt install xubuntu-desktop blender` inside of Ubuntu Noble. 

Thanks for anything.

**Comment by anonimo82** [comment]
P.S.: Did you disable the phantom process killer already?

**Comment by anonimo82** [comment]
@ihtarlik Sorry, I forgot to mention: this is also for me to check out if I need to buy a new device or if I can use the one I already have.
I have a Samsung Galaxy S20 5G, and I can't run Blender in XUbuntu Noble with VirGL, but I don't know if it's a misconfiguration issue or if it's an hardware one. If you tell me if you made it to run Blender with your device, I'll know I can do it with a newer device, and I'll act accordingly buying a new one.

So, please, will you help me with this?
Thanks in advance for either, I don't want to bother you too much, you already did what you can.

Regards, have a nice day.

**Comment by ihtarlik** [comment]
@anonimo8 Yes, I disabled the phantom process killer. In order to run the basic desktop for more than a few seconds, I had to do so.

I cannot get Ubuntu+Gnome to run. However, I am running Debian+LXDE, and I am able to run Blender without an issue. It's slow, but it runs. And this is on a a Samsung A14 5G, which is slower than an S20. The A14 is running a MediaTek chipset, while the S20 has Qualcomm, but they both appear to be running a Mali graphics chip. So, I believe you to have a configuration problem, not a hardware sufficiency problem.

Try running a different version of Linux in PROOT. Also, you can use GLXGears from the mesa-utils package to test 3D acceleration to verify that separately from Blender.

**Comment by anonimo82** [comment]
Yes, I see. I also found almost the same info in the meantime, check this issue: https://github.com/LinuxDroidMaster/Termux-Desktops/issues/68

I have found the potential issue is in Zink configuration.

Please, excuse me for the disturb and the confusion.
